### PR TITLE
Bugfix/#76: Excluded links that are already handled by the "External Links" (New Window) library

### PR DESCRIPTION
**Pull request by kzenmatthias**
The script adds the attribute `data-extlink` to links where icons are being added. We can use this to exclude links with already have an icon.

### CODE REVIEW

File: js/sources/extlink-window.js [hunk 1]
```diff
@@ -159,7 +159,9 @@
   Drupal.behaviors.extlinkWindow = {
     // To understand behaviors, see https://www.drupal.org/node/2269515
     attach: function (context, drupalSettings) {
-      var links =  context.querySelectorAll('a[href]:not([href^="#"]):not([href^="tel:"]):not([role="tab"]):not([role="button"])');
+      var links =  context.querySelectorAll(
```
Comment (GyD) [inline js/sources/extlink-window.js:162]: just a test

**Review by GyD — COMMENTED** [review]
test

**Review by smillart — CHANGES_REQUESTED** [review]

File: js/sources/extlink-window.js [hunk 2]
```diff
@@ -159,7 +159,9 @@
...
+        'a[href]:not([href^="#"]):not([href^="tel:"]):not([role="tab"]):not([role="button"]):not(data-extlink)'
```
[the remark this answers is not in the document]
Comment (kzenmatthias) [reply js/sources/extlink-window.js]: I've added data-extlink-window and ignored this attribute in the selector.
Changes can be found in the new commit.

**Review by smillart — COMMENTED** [review]
I need to test the branch on my local. I'll keep you posted...

**Review by smillart — APPROVED** [review]
It seems your modifications no longer produce any JS error in the console! All `<a>` elements with the `data-extlink-window` attribute are safely ignored. I approved the PR... Congrats!

**Pull request merged**